docs/fedora-releases-guide/dist-git-onboarding.md

```diff
@@ -88,8 +88,7 @@ and not from `upstream_project_url`.
 You can check the other customization options [here](./index.md#customization).
 
 :::info Divergent dist-git branches
-Current default behaviour of the release syncing results in having divergent dist-git branches. If you want to avoid this,
-please see the details [here](/docs/fedora-releases-guide#keeping-dist-git-branches-non-divergent).
+By default, release syncing can result in divergent dist-git branches. To prevent this, refer to the details [here](/docs/fedora-releases-guide/non-divergent-dist-git-branches.md).
```

Comment:
```suggestion
By default, release syncing can result in divergent dist-git branches. To prevent this, refer to the configuration details [here](/docs/fedora-releases-guide/non-divergent-dist-git-branches.md).
```